### ISSUE TITLE
Current CDS instance shuts down on 26/09/2024

### ISSUE THREAD

**Issue by fwimp** [pearselab/areadata#15]
It looks like the current instance of CDS shuts down on 26/09/2024. As such `cds-era5-args.py` may break as the underlying package `cdsapi` may not initially work. 

https://forum.ecmwf.int/t/informative-page-in-readiness-for-new-cds-beta/2985/28

It looks like the new package for interfacing with the api _may_ be https://github.com/ecmwf-projects/cads-api-client

See this python notebook for usage
https://github.com/ecmwf-projects/cads-api-client/blob/main/notebooks/cads_api_client_test.ipynb

Just making sure this is on the radar, as it might cause problems fairly quickly, and the changeover will happen this month.

**Comment by fwimp** [comment]
Yeah I'm not clicking on that link, and I'm 95% sure that is spam so I recommend not clicking on it either.

**Comment by willpearse** [comment]
@smithtp I feel like we've done a good job of maintaining this for a while, and it seems like this is going to break the resource unless we fix it. @fwimp , does it fall within scope of UK VBD Hub to fix this? It feels to me like it does, but of course I would say that... It also doesn't look like this would be absolutely awful to fix, as I think the endpoints are essentially the same across the two versions (right?)

**Comment by willpearse** [comment]
(also thank you for spotting this, Francis, I appreciate it!)

**Comment by smithtp** [comment]
Turns out this seems to be easily resolvable as the new cds api works much the same as the old one, just needed a new API key and to accept the relevant data licenses on the new CDS site.
